tests/test_close_trustline.py

```diff
@@ -47,14 +47,15 @@ def currency_network_contract_with_trustlines(chain, web3, accounts, interest_ra
             currency_network_contract.functions.setAccount(
                 a,
                 b,
-                1000000,  # creditline given
-                1000000,  # creditline received
-                interest_rate,  # interest rate given
-                interest_rate,  # interest rate received
-                0,  # fees outstanding a
-                0,  # fees outstanding b
```

Comment:
To me these comments are useful.Why did you remove them?

I would prefer if we always provide keyword arguments, when calling smart contracts from python. I know this isn't true for most of this file.

Reply:
Because I used `black -l 1000` to use a regex to modify function signatures without having to bother with line breaks and it fucked this comment, then I was too lazy to remove it since I did not feel responsible for black fucking it up.